examples/common/python/database/lmdb_helper_proxy.py

```diff
@@ -167,3 +175,60 @@ def __escape(self, string):
 # ------------------------------------------------------------------------------
     def __unescape(self, string):
         return string.encode("utf-8").decode("unicode_escape")
+
+
+class TextServiceClient(object):
```

Comment:
This change (moving TextServiceClient to this file) should be separate PR

Reply:
Anyway it has to be separated out and this PR is separate our client specific modules. Client won't use it and specific lmdb_helper_proxy.